datafusion/common/src/table_reference.rs

```diff
@@ -189,7 +234,8 @@ impl<'a> TableReference<'a> {
         }
     }
 
-    /// Converts directly into an [`OwnedTableReference`]
+    /// Converts directly into an [`OwnedTableReference`] by copying
```

Comment:
copying -> cloning maybe?

not sure how explicit need to be about cloning in documentation